02-Reproject-Raster.Rmd

```diff
@@ -203,7 +203,6 @@ plot(DTM,
      legend=F)
 ```
 
```

Comment:
EDIT: 
We have now successfully layered the Digital Terrain Model on top of our hillshade
to produce a nice looking, textured basemap!